src/phoenix/config.py

```diff
@@ -74,9 +74,18 @@
 ENV_PHOENIX_SECRET = "PHOENIX_SECRET"
 ENV_PHOENIX_API_KEY = "PHOENIX_API_KEY"
 ENV_PHOENIX_USE_SECURE_COOKIES = "PHOENIX_USE_SECURE_COOKIES"
+ENV_PHOENIX_PASSWORD_RESET_TOKEN_EXPIRY = "PHOENIX_PASSWORD_RESET_TOKEN_EXPIRY"
 ENV_PHOENIX_ACCESS_TOKEN_EXPIRY = "PHOENIX_ACCESS_TOKEN_EXPIRY"
 ENV_PHOENIX_REFRESH_TOKEN_EXPIRY = "PHOENIX_REFRESH_TOKEN_EXPIRY"
 
+ENV_PHOENIX_SMTP_ENABLED = "PHOENIX_SMTP_ENABLED"
```

Comment:
I would nix this. Having the below values set should be enough

Reply:
OK